stemcell_builder/stages/untar_base_os_image/apply.sh

```diff
@@ -6,7 +6,7 @@ base_dir=$(readlink -nf $(dirname $0)/../..)
 source $base_dir/lib/prelude_apply.bash
 
 # the tar_base_os_image stage creates a tar ball of the chroot with no enclosing directory
-tar zxf $os_image_tgz -C $chroot
+sudo tar zxf $os_image_tgz -C $chroot
```

Comment:
Why is this `sudo` necessary?

Reply:
I forget why. I'll look at this as well tomorrow.